pipelines/data_pipelines.go

```diff
@@ -0,0 +1,43 @@
+package pipelines
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/pipelines"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourcePipelines() *schema.Resource {
+	type pipelineData struct {
+		Name string            `json:"pipeline_name,omitempty"`
+		Ids  map[string]string `json:"ids,omitempty" tf:"computed"`
+	}
+	return common.WorkspaceData(func(ctx context.Context, data *pipelineData, w *databricks.WorkspaceClient) error {
+		errorMessage := fmt.Errorf("no pipelines found")
+		pipelineSearch := pipelines.ListPipelines{MaxResults: 100}
+
+		if data.Name != "" {
+			searchPattern := fmt.Sprintf("name LIKE '%s'", data.Name)
+			errorMessage = fmt.Errorf("there is no pipeline with name LIKE '%s'; you need to specify `pipeline_name` as an `exact_name` or with percent wildcards", data.Name)
```

Comment:
replace 

```
`exact_name` 
```

with something like "full name"?

Reply:
adjusted text from "as an `exact_name`" to "as the full pipeline name"